build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -182,8 +184,9 @@ CONFIG_BOARD_LOOPSPERMSEC=22222
 # Interrupt options
 #
 CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
-CONFIG_ARCH_INTERRUPTSTACK=0
-# CONFIG_ARCH_HAVE_NESTED_INTERRUPT is not set
+CONFIG_ARCH_INTERRUPTSTACK=1024
```

Comment:
Is using interrupt stack mandantory?

Reply:
According to the Kconfig it is mandatory.